ports/nrf/boards/feather52840/README.md

```diff
@@ -197,19 +188,19 @@ Device programmed.
 
 ### Flashing CircuitPython with MSC UF2
 
-Make `uf2` target to generate the uf2
+uf2 file is generated at the end of `all` target
 
 ```
-$ make V=1 SD=s140 SERIAL=/dev/tty.usbmodem1411 BOARD=feather52840 all uf2
+$ make V=1 SD=s140 SERIAL=/dev/tty.usbmodem1411 BOARD=feather52840 all
 Create firmware.uf2
-../../tools/uf2/utils/uf2conv.py -c -o "build-feather52840-s140/firmware.uf2" "build-feather52840-s140/firmware.hex"
+../../tools/uf2/utils/uf2conv.py -f 0xADA52840 -c -o "build-feather52840-s140/firmware.uf2" "build-feather52840-s140/firmware.hex"
 Converting to uf2, output size: 392192, start address: 0x26000
 Wrote 392192 bytes to build-feather52840-s140/firmware.uf2.
 ```
 
 Simply drag and drop firmware.uf2 to the MSC, the nrf52840 will blink fast and reset after done.
 
-**Note**: you need to update `tools/uf2` for uf2conv.py to support hex file input, current circuitpython's master use older verion of uf2conv.py which only support biin file input. To update, change directory to top folder of circuitpython and run. The size of uf2 should be ~400KB, if using the old uf2conv.py the output file would be 1 MB which is not correct.
+**Note**: you need to update `tools/uf2` for uf2conv.py to support hex file input, current circuitpython's master use older verion of uf2conv.py which only support bin file input. To update, change directory to top folder of circuitpython and run. The size of uf2 should be ~400KB, if using the old uf2conv.py the output file would be 1 MB which is not correct.
```

Comment:
Is this still true? we should update uf2conv.py so this doesn't need to be done by hand.

Reply:
yeah, it is updated, will remove this